test/key.spec.js

```diff
@@ -3,16 +3,14 @@
 'use strict'
 
 const expect = require('chai').expect
-const path = require('path')
 
 const Key = require('../src').Key
 
-const pathSep = path.sep
-const n = (p) => path.normalize(p)
+const pathSep = '/'
```

Comment:
If this is defined in multiple places, it would be better to have a `src/constants.js` file

Reply:
I'm not happy with the names, just dealing with the current code.  You will note that `pathSep` is a Buffer in one and a string in another!

Reply:
You are welcome to change that :)

Reply:
@richardschneider you should be able to fetch this branch and push commits to it.